src/components/Loader.vue

```diff
@@ -0,0 +1,307 @@
+<!--
+Usage:
+
+<Loader
+    :title="isLoading ? 'Loading things' : 'Things done'"
```

Comment:
The external component is free to use whichever means they want to get a title in there. But I think in the use cases that we currently have, there is only the loading -> success case.

Reply:
But in e.g. the direct error case, you also show the title. If the example is just for loading->success case, we don't need message at all. That's what I don't like about it. Either we should put a real example or just go with :title='the current title' and so on.

Reply:
yes. It will end up being a computed property most of the times. 
I don't think too much emphasis on that is required, as that is caller side. The component itself does not distinguish the title for the different state values.